tests/test_gpu_examples.py

```diff
@@ -1378,3 +1379,87 @@ def test_model_loaded_in_float16_working(self):
                 data_collator=DataCollatorForLanguageModeling(self.tokenizer, mlm=False),
             )
             trainer.train()
+
+
+@require_torch_gpu
+@require_auto_awq
+class PeftAwqGPUTests(unittest.TestCase):
+    r"""
+    Awq + peft tests
+    """
+
+    def setUp(self):
+        self.causal_lm_model_id = "ybelkada/opt-125m-awq"
```

Comment:
Let's try to use https://huggingface.co/peft-internal-testing for testing artifacts in the future.

Reply:
ok I'll upload it there

Reply:
Done!